spec.html

```diff
@@ -378,6 +378,20 @@ <h1>GrowableSharedArrayBuffer.prototype [ @@toStringTag ]</h1>
 <emu-clause id="sec-properties-of-the-arraybuffer-prototype-object-mods">
   <h1>Modifications to Properties of the ArrayBuffer Prototype Object</h1>
 
+  <emu-clause id="sec-get-arraybuffer.prototype.bytelength">
+    <h1>get ArrayBuffer.prototype.byteLength</h1>
+    <p>`ArrayBuffer.prototype.byteLength` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:</p>
+    <emu-alg>
+      1. Let _O_ be the *this* value.
+      1. Perform ? RequireInternalSlot(_O_, [[ArrayBufferData]]).
+      1. If IsSharedArrayBuffer(_O_) is *true*, throw a *TypeError* exception.
+      1. <ins>If IsResizableArrayBuffer(_O_) is *true*, throw a *TypeError* exception.</ins>
```

Comment:
why is this throwing instead of reporting the current size? or is the idea that there is no current size?

Reply:
This is to make sure the existing `ArrayBuffer.prototype.byteLength` remains non-generic (it already doesn't work on SABs, for example). `ResizableArrayBuffer` has a separate prototype.

Reply:
ahh, gotcha, makes sense.